include/mesh/mesh_tetgen_interface.h

```diff
@@ -155,6 +159,12 @@ class TetGenMeshInterface
   std::vector<unsigned> _sequential_to_libmesh_node_map;
 
   /**
+   * Parameter controling the behaviour of tetgen.
+   * By default quiet.
+   */
+  std::string _switches = "Q";
```

Comment:
Oh, probably because we don't test with `--disable-strict-lgpl`, so this new code is not even being tested.